run_tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+RUST_LOG=info go test -count=1 ./...
```

Comment:
Suggestion: Run the Rust tests here as well.

Reply:
Updated.  But bad news, rust tests are failing.

Reply:
Some of which I think are just missing params.  But some of which are not.  Any ideas @laser ?

Reply:
> Any ideas @laser ?

Not offhand, no. I'll check the branch out and give it a test.